Documentation/docpage-css/navigation.css

```diff
@@ -1,23 +1,60 @@
 /* horizontal nav bar */
 
-.logo {
-    color: var(--primary-color);
-    margin-left: 0.8rem;
-    font-size: 1.6rem;
-}
-
-.nav-header {
+.nav-header,
+.desktop-navigation-bar {
     grid-area: nav-header;
     position: sticky;
     top: 0;
-    padding: 0.7rem 1rem;
+    padding: 0 1rem 0rem;
     z-index: 2;
+    height: 4rem;
     display: flex;
     align-items: center;
     justify-content: space-between;
     font-weight: 500;
     background-color: var(--primary-bg);
-    box-shadow: 0 10px 8px -8px var(--btn-hover);
+}
+
+.nav-header-desktop,
+.nav-header-mobile{
+    box-shadow: 0px 3px 8px var(--btn-hover);
+}
+
+.desktop-navigation-bar {
+    box-shadow: 0px 3px 8px var(--modal-shadow);
+    min-width: 32.5rem;
+}
+
+.nav-header-desktop>a>img,
+.nav-header-mobile>div>a>img,
+.logo {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 10rem;
+}
+
+.nav-header-desktop > h2 {
+    font-size: 1.2rem;
+}
+
+.nav-header-mobile>div>a>.logo-icon{
+    width: 1.6rem;
+}
+
+.desktop-navigation-bar>[class*="-search-input"] {
+    flex-grow: 1;
+    margin: 0 1rem 0 2rem;
+}
```

Comment:
You used CSS selectors extensively which is great if you don't want to create irrelevant CSS classes. UI is super clean. Files are clean and readable. Great work. Go ahead and merge the branch